api/src/test/java/org/pmiops/workbench/api/ProfileControllerTest.java

```diff
@@ -58,296 +54,304 @@
-    verify(fireCloudService).registerUser(null, GIVEN_NAME, FAMILY_NAME);
-    verify(fireCloudService).createAllOfUsBillingProject(projectName);
-    verify(fireCloudService).addUserToBillingProject(PRIMARY_EMAIL, projectName);
-  }
-
-  @Test
-  public void testMe_userBeforeSuccessCloudProjectConflict() throws Exception {
-    createUser();
-    User user = userDao.findUserByEmail(PRIMARY_EMAIL);
-    when(userProvider.get()).thenReturn(user);
-    when(fireCloudService.isRequesterEnabledInFirecloud()).thenReturn(true);
-
-    String projectName = BILLING_PROJECT_PREFIX + user.getUserId();
-    doThrow(new ApiException(HttpStatus.CONFLICT.value(), "conflict"))
-        .when(fireCloudService).createAllOfUsBillingProject(projectName);
-    doThrow(new ApiException(HttpStatus.CONFLICT.value(), "conflict"))
-        .when(fireCloudService).createAllOfUsBillingProject(projectName + "-1");
-
-    Profile profile = cloudProfileController.getMe().getBody();
-
-    // When a conflict occurs in dev, log the exception but continue.
-    assertProfile(profile, PRIMARY_EMAIL, CONTACT_EMAIL, FAMILY_NAME, GIVEN_NAME,
-        DataAccessLevel.UNREGISTERED, TIMESTAMP, projectName + "-2",
-        true);
-    verify(fireCloudService).registerUser(CONTACT_EMAIL, GIVEN_NAME, FAMILY_NAME);
-    verify(fireCloudService).createAllOfUsBillingProject(projectName);
-    verify(fireCloudService).createAllOfUsBillingProject(projectName + "-1");
-    verify(fireCloudService).createAllOfUsBillingProject(projectName + "-2");
-    verify(fireCloudService).addUserToBillingProject(PRIMARY_EMAIL, projectName + "-2");
-  }
-
-  @Test
-  public void testMe_userBeforeSuccessCloudProjectTooManyConflicts() throws Exception {
-    createUser();
-    User user = userDao.findUserByEmail(PRIMARY_EMAIL);
-    when(userProvider.get()).thenReturn(user);
-    when(fireCloudService.isRequesterEnabledInFirecloud()).thenReturn(true);
-
-    String projectName = BILLING_PROJECT_PREFIX + user.getUserId();
-    doThrow(new ApiException(HttpStatus.CONFLICT.value(), "conflict"))
-        .when(fireCloudService).createAllOfUsBillingProject(projectName);
-    doThrow(new ApiException(HttpStatus.CONFLICT.value(), "conflict"))
-        .when(fireCloudService).createAllOfUsBillingProject(projectName + "-1");
-    doThrow(new ApiException(HttpStatus.CONFLICT.value(), "conflict"))
-        .when(fireCloudService).createAllOfUsBillingProject(projectName + "-2");
-    doThrow(new ApiException(HttpStatus.CONFLICT.value(), "conflict"))
-        .when(fireCloudService).createAllOfUsBillingProject(projectName + "-3");
-    doThrow(new ApiException(HttpStatus.CONFLICT.value(), "conflict"))
-        .when(fireCloudService).createAllOfUsBillingProject(projectName + "-4");
-
-    try {
-      cloudProfileController.getMe();
-      fail("ServerErrorException expected");
-    } catch (ServerErrorException e) {
-      // expected
+    private static final Instant NOW = Instant.now();
```

Comment:
The changes in this file are mostly a conversion from two spaces (correct) to four spaces (incorrect). Is your editor configured correctly?